internal/ingress/metric/collectors/socket.go

```diff
@@ -249,6 +251,10 @@ func (sc *SocketCollector) handleMessage(msg []byte) {
 			continue
 		}
 
+		if sc.reportStatusClasses && len(stats.Status) > 0 {
+			stats.Status = fmt.Sprintf("%cxx", stats.Status[0])
```

Comment:
Should we use another attribute like `StatusClass` for this?

Reply:
We could do it, the only downside is that it might break existing dashboards and queries.